Sources/Tuvi.Core.Mail.Impl/Commands.cs

```diff
@@ -111,9 +111,8 @@ protected virtual string GetUniqueCommandIdentifier(string _)
 
         protected virtual bool IsHighPriority { get => false; }
 
-        public async Task<T> RunCommand(string email, ICredentialsProvider credentialsProvider, CancellationToken cancellationToken)
+        public async Task<T> RunCommand(string email, CancellationToken cancellationToken)
```

Comment:
Rename `RunCommand` to `RunCommandAsync`

Reply:
Done.